crates/node/builder/src/components/mod.rs

```diff
@@ -49,7 +50,9 @@ pub trait NodeComponents<T: FullNodeTypes>: Clone + Unpin + Send + Sync + 'stati
     type Consensus: FullConsensus<<T::Types as NodeTypes>::Primitives> + Clone + Unpin + 'static;
 
     /// Network API.
-    type Network: FullNetwork;
+    type Network: FullNetwork<
```

Comment:
same q, seems useful to have bound on Broadcasted/Pooled tx here?

Reply:
ah I guess those are not useful because most of the network setup is happening in BuilderContext